elastic/eui
# High contrast mode

## Summary

closes https://github.com/elastic/eui/issues/8133
See https://github.com/elastic/eui/issues/7509#issuecomment-2260692852

### Screenshots

<img width="1484" alt="" src="https://github.com/user-attachments/assets/76d35c25-9af7-4555-871f-2b9e1cfa1ddd">

<img width="828" alt="" src="https://github.com/user-attachments/assets/277efe79-a837-4bbb-9d7c-19ee703c1a65">

<img width="1486" alt="" src="https://github.com/user-attachments/assets/49fab9c7-e15f-4623-bf8e-8d21464a9d4d">

### Extra reading/reference:

-  https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast
- https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors
- https://kilianvalkhof.com/2023/css-html/i-no-longer-understand-prefers-contrast/


## QA

- [MacOS] Go to Display > Increase contrast and check the toggle
  <img width="488" alt="" src="https://github.com/user-attachments/assets/084e4aee-7d15-44e6-9b42-e5537cba35dc">
- Go to https://eui.elastic.co/pr_8036/#/

### General checklist

- Browser QA
      - (_[emulate forced colors](https://devtoolstips.org/tips/en/emulate-forced-colors/) if you do not have access to a Windows machine_.)
- Docs site QA
    -~ [ ] Checked **[Code Sandbox](https://codesandbox.io/)** works for any docs examples
- Code quality checklist
    ~- [ ] Added or updated **[jest](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/unit-testing.md) and [cypress](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/cypress-testing.md) tests**~
- Release checklist
    ~- [ ] If applicable, added the **breaking change** issue label (and filled out the breaking change checklist)~
- Designer checklist - N/A
